src/app/Dashboard/LayoutTemplateGroup.tsx

```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { dashboardConfigTemplateHistoryClearIntent } from '@app/Shared/Redux/ReduxStore';
```

Comment:
We still need this. Just remove all changes for this file.

https://github.com/cryostatio/cryostat-web/blob/5aa3f5c99bbec8f6cd58d8bfce5751c0dbbb7cd1/src/app/Dashboard/LayoutTemplateGroup.tsx#L83